### PR TITLE
model: scikit: scorer: Add scikit scorers 

### LINKED ISSUE #733: accuracy: scikit: Wrap sklearn.metrics accuracy methods within Accuracy plugins
Add these to `model/scikit/dffml_model_scikit/accuracy/`

https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html

Depends: #732

### PR DESCRIPTION
**Pull request by sk-ip**
closes #733

Signed-off-by: Sudhanshu Sudhanshukumar5459@gmail.com

### CODE REVIEW

**Comment by pdxjohnny** [comment]
- [ ] Have testcases pull variables and subclass from original testcase class
- [x] Provide config for each scorer
